### PR TITLE
Add more references between lowercase/uppercase operations.

### PR DESCRIPTION
**Pull request by frewsxcv**

### CODE REVIEW

File: src/libstd/ascii.rs [hunk 1]
```diff
@@ -77,6 +82,9 @@ pub trait AsciiExt {
     /// assert_eq!('A', ascii.to_ascii_uppercase());
     /// assert_eq!('❤', utf8.to_ascii_uppercase());
     /// ```
+    ///
+    /// [`make_ascii_uppercase`]: #tymethod.make_ascii_uppercase
+    /// [`str::to_uppercase`]: .../primitive.str.html#method.to_uppercase
```
Comment (GuillaumeGomez) [inline src/libstd/ascii.rs]: "..."? ;)

File: src/libstd/ascii.rs [hunk 2]
```diff
@@ -96,6 +109,9 @@ pub trait AsciiExt {
     /// assert_eq!('a', ascii.to_ascii_lowercase());
     /// assert_eq!('❤', utf8.to_ascii_lowercase());
     /// ```
+    ///
+    /// [`make_ascii_lowercase`]: #tymethod.make_ascii_lowercase
+    /// [`str::to_lowercase`]: .../primitive.str.html#method.to_lowercase
```
Comment (GuillaumeGomez) [inline src/libstd/ascii.rs]: "..."

**Comment by GuillaumeGomez** [comment]
Nice idea! r=me once urls fixed.

**Comment by frewsxcv** [comment]
@bors r+

🤖 🙅‍♂️

**Comment by frewsxcv** [comment]
@bors r=GuillaumeGomez 

Woops, didn't mean to r+ myself

**Pull request merged**